bodhi/server/schemas.py

```diff
@@ -522,6 +522,12 @@ class ListUpdateSchema(PaginatedSchema, SearchableSchema, Cosmetics):
         missing=None,
     )
 
+    current_release = Releases(
```

Comment:
I wonder if "active_release" would be more appropriate? This is OK too, just trying to think of a word that might more clearly map to the behavior.

Reply:
Let's also pluralize this, so "active_releases".